### ISSUE TITLE
Project Structure

### ISSUE THREAD

**Issue by austek** [Swagger2Markup/swagger2markup-maven-plugin#54]
Project structure is confusing, it seems to be a maven project but settings in pom seems to be old and tests in `src/it` folder seems to be out-of-date and not run. There are is gradle project setup in place as well

**Comment by abelsromero** [comment]
I agree but I don't critic the solution nor the decisions behind it.
Just I think that adding some notes in the README explaining how to setup the project and start hacking would be nice.

**Comment by RobWin** [comment]
Yes, I agree. It is just too much effort for me to maintain everything by my own. In my spare time I focos on [resilience4j ](https://github.com/resilience4j/resilience4j )at the moment.
I need more support by the community.

**Comment by austek** [comment]
What build tool is the project supposed to be, I might be able to help, or for anyone else who wants to help.
